homeassistant/components/uptimerobot/sensor.py

```diff
@@ -61,6 +60,8 @@ async def async_setup_entry(
 class UptimeRobotSensor(UptimeRobotEntity, SensorEntity):
     """Representation of a UptimeRobot sensor."""
 
+    _attr_name = None
```

Comment:
Should this not be set on `UptimeRobotEntity` and `name` removed from the `BinarySensorEntityDescription` definition in `homeassistant/components/uptimerobot/binary_sensor.py`?

Reply:
Oh whoops, wrong file then

Reply:
We could do that, but then the switch loses it's name (active) (which is also fine by me)